.github/ISSUE_TEMPLATE/bug-report-advanced

```diff
@@ -0,0 +1,74 @@
+name: "\U0001F41E Bug report"
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before submitting a bug report**
+
+        The issue list is reserved for bug reports and feature requests. If you have a usage question, you can:
+
+        - Read the [documentation](https://blazorise.com/docs)
+        - Ask questions on [Discord](https://discord.io/blazorise)
```

Comment:
Worth adding the GH Discussion tab for people who do not use Discord, but still want to write a custom question?